File: Mysearch.py
def search(mysearch,num):
    ch = len(mysearch) - 1
    for i in range(0, len(mysearch)):
        if mysearch[i] == num:
            print(num, " was found in Array")
        else:
            ch = ch - 1
    if ch == -1:
        print("Number not found in Array")
    return

# this is an example of the function to search a number in 1,2,5,6,3,4 etc
mysearch = [1,2,5,6,3,4,77,8,9,11,10,-8]
num = (int(input("Enter a number to search that in you Array: ")))
print(search(mysearch, num))

File: Mysearch.java
import java.util.Scanner;
public class Mysearch {
    // The following is the main, contains the array and the input for a number to search.
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Please enter a number you want to search for in your array: ");
        int num = input.nextInt();
        int [] mysearch = {1,2,5,6,3,4,77,8,9,11,10,-8};
        mySearch(mysearch,num);
    }
    // The following is a method, outputs the result
    public static void mySearch(int [] mysearch,int num) {
        int ch = mysearch.length - 1;
        for (int i = 0;i < mysearch.length; i++) {
            if (mysearch[i] == num){
            System.out.print(num);
            System.out.print(" was found in the Array");
            }else{
            ch = ch - 1;
            }
        }
        if (ch == -1) {
            System.out.print("Number not found in Array");
        }
    }
}
